<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfacciaUtente</class>
 <widget class="QWidget" name="InterfacciaUtente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:rgb(113, 113, 113);
}

QPushButton
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0.507177 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 127, 255));
	font: bold 14pt &quot;Kinnari&quot;;
	color: rgb(255, 170, 0);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;
}

QPushButton:hover
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.507177 rgba(255, 170, 0, 255)); 
	color: rgb(0, 0, 0);
}


QPushButton:hover:!pressed
{
	background-color:qlineargradient(spread:pad, x1:0.244, y1:1, x2:0.837, y2:0.227273, stop:0 rgba(255, 255, 127, 255), stop:0.507177 rgba(0, 0, 0, 255));
	color: rgb(255, 170, 0);
}

#BenvenutoLabel
{
	font: bold italic 36pt &quot;Kinnari&quot;;
	color: rgb(255, 85, 0)
}

#ProfiloLabel
{

	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0)
}

#AmiciLabel
{

	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0)
}

#RicercaLabel
{

	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0)
}

QLabel
{
	font: bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
}

QLineEdit
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 170, 0);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

QTableWidget
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 255);
	background-color:rgb(35, 35, 35);
	gridline-color:rgb(255, 85, 0);
}

QHeaderView::section 
{
	padding:0;
	height: 32px;
	width:18px;
	font:bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 85, 0);
	background-color:rgb(118, 118, 118);
}

QTableWidget::item:selected
{ 
	color: rgb(0, 0, 0);
	background-color: rgb(255, 170, 0);
 }

QTableWidget QTableCornerButton::section 
{
	background-color:rgb(255, 85, 0);
}

QGroupBox
{
	background-color:rgb(57, 57, 57);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;

}

#Username1Label
{
	background-color:rgb(57, 57, 57);
}

#Username2Label
{
	background-color:rgb(57, 57, 57);
}

#Username3Label
{
	background-color:rgb(57, 57, 57);
}

#LogoutButton
{
	font: bold 16pt &quot;Kinnari&quot;;
}
</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>720</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="BenvenutoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>680</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kinnari</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ProfiloLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Profilo Personle</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>720</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="VisualizzaProfiloButton">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>90</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Visualizza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModificaProfiloButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Modifica</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>218</y>
     <width>330</width>
     <height>88</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="AggiungiAmicoLineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="AggiungiAmicoButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Aggiungi</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Username1Label">
      <property name="text">
       <string>Usename</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="EliminaAmicoLineEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="EliminaAmicoButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Elimina</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Username2Label">
      <property name="text">
       <string>Usename</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>720</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>155</y>
     <width>10</width>
     <height>170</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>220</y>
     <width>330</width>
     <height>88</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="CercaLineEdit">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>35</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CercaButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>35</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Cerca</string>
    </property>
   </widget>
   <widget class="QLabel" name="Username3Label">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>5</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Usename</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="VisualizzaAmiciButton">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>400</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Visualizza Amici</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VisualizzaUtentiIscrittiButton">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>480</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Visualizza Tutti Gli Utenti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoutButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>600</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Table">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>350</y>
     <width>322</width>
     <height>230</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cognome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="AmiciLabel">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>170</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Amici</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>---------&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>---------&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>oppure</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RicercaLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ricerca</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
